src/flag_gems/runtime/__init__.py

```diff
@@ -21,4 +21,8 @@ def get_triton_config(op_name):
     return config_loader.get_triton_config(op_name)
 
 
+def get_heuristics_config(op_name):
```

Comment:
what about renaming them as get_tuned_config and get_heuristic_config?

Reply:
Done